packages/for-ui/src/chip/Chip.stories.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
-import { Meta, Args } from '@storybook/react/types-6-0';
-import { MdCheck } from 'react-icons/md';
+import { Meta } from '@storybook/react/types-6-0';
 
 import { Text } from '../typography/Typography';
 import { Chip } from './Chip';
+import { BsPaperclip } from 'react-icons/bs';
```

Comment:
MdAttachFileをつかってください！

```suggestion
import { MdAttachFile } from 'react-icons/md'
```